src/Form/Fields.php

```diff
@@ -36,19 +37,52 @@ public function __set(string $name, $field): void
 		parent::__set($field->name(), $field);
 	}
 
+	/**
+	 * Returns an array with the default value of each field
+	 */
+	public function defaults(): array
+	{
+		return $this->toArray(fn ($field) => $field->default());
+	}
+
+	/**
+	 * An array of all found in all fields errors
+	 */
+	public function errors(): array
```

Comment:
We have been caching the errors in the Form class. Do you think it's ok regarding performance to not cache them here?

Reply:
You are right. We should still cache them. I just need to make sure that the cache gets flushed when the fill method is used.

Reply:
☑️